package.json

```diff
@@ -23,7 +23,8 @@
     "gulp-jsdoc-to-markdown": "^1.1.1",
     "gulp-mocha": "^2.1.3",
     "gulp-rename": "^1.2.2",
-    "promise": "^7.0.4",
+    "path-exists": "^3.0.0",
+    "pinkie": "^2.0.4",
```

Comment:
Nah, I would like to keep node for now, since parse5 is still used in some browsers without proper ES6 support, also node 10 is still widely used. I would like to make sure that we don't break anything by using ES6 syntax